pyproject.toml

```diff
@@ -29,4 +31,23 @@ addopts = "-ra --cov"
 testpaths = [
     "tests"
 ]
-pythonpath = ['.']
+pythonpath = ['src']
+
+
+# content of: tox.ini , put in same dir as setup.py
```

Comment:
The comment looks like copypasta? (Since this repo has no setup.py)

Reply:
It is and should be removed... removing